Makefile

```diff
@@ -55,17 +55,20 @@ link-development-env:
 run: build link-development-env
 	pipenv run flask run
 
-run-gunicorn: link-development-env
-	WEB_SERVER=gunicorn pipenv run ./run_app.sh
+run-gunicorn-gevent: link-development-env
+	WEB_SERVER_TYPE=gunicorn WEB_SERVER_WORKER_CLASS=gevent pipenv run ./run_app.sh
+
+run-gunicorn-gthread: link-development-env
```

Comment:
What do you think about making an assumption that gRPC will always be on when using gthread?

Reply:
Could do, what about uwsgi?. Also, for dev, it shouldn't make much of a difference.

Reply:
uWSGI as well